pkg/utils/configs/config.go

```diff
@@ -38,9 +44,8 @@ connector {
     name: {{.Name}}-connector
     host: {{.Host}}
     port: {{.Port}}
-    dataConnectionCount: {{.Dcc}}
     role: {{.Role}}
-    cost: {{.Cost}}
+    cost: {{.Cost}} {{if .ConCount}}{{printf "\n    dataConnectionCount:" }} {{.ConCount}}{{end}}
```

Comment:
Noting the comment above, I don't think this is the right way to handle the issue. Use `{{- if .ConCount}} ... {{- end}}` as in QdRouterConfig() method.